app/views/plantings/show.html.haml

```diff
@@ -28,11 +28,13 @@
           = "Planted from: "
         = "#{display_planted_from(@planting)}"
     
-    - if !@planting.sunniness.blank?
-      %p
-        %b 
-          = "Sun or shade?: "
-        = "#{display_sunniness(@planting)}"
+    %p
+      %b 
+        = "Sun or shade?: "
+      - sunniness = @planting.sunniness.blank? ? "not specified" : @planting.sunniness
+      = image_tag("sunniness_#{sunniness}.png", :size => "25x25", :alt => "#{sunniness}", :title => "#{sunniness}")
+      = " (#{sunniness})"
```

Comment:
Great, and once people are used to the changes, we can eventually decide if we want to keep the text or one day remove it.